commit 3b2a1e2bb5e28c8c31adc7559ce94c9d3a5a3d7a (refs/changes/60/3760/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-04-03T10:46:13-07:00 (1 year, 6 months ago)
    
    TRITON-288 sdcnode updates for March 2018 node.js security releases
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index d0856f3..c0342f2 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ JSSTYLE_FLAGS	 = -f tools/jsstyle.conf
 SMF_MANIFESTS_IN = smf/manifests/docker.xml.in
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v4.8.7
+NODE_PREBUILT_VERSION=v4.9.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
